common/src/main/java/whocraft/tardis_refined/common/capability/TardisLevelOperator.java

```diff
@@ -165,12 +166,11 @@ public void setInitiallyGenerated(boolean hasInitiallyGenerated) {
 
     /**
      * Moves the entity into the TARDIS. If the TARDIS has no door established, the entity is sent to 0,100,0.
-     *
      **/
     public boolean enterTardis(Entity entity, BlockPos externalShellPos, ServerLevel shellLevel, Direction shellDirection) {
 
         if (!entity.level().isClientSide()) {
-            if (this.level instanceof ServerLevel targetServerLevel){
+            if (this.level instanceof ServerLevel targetServerLevel) {
```

Comment:
When pulling a instanceof, you do not need to check if we are on the client because the instanceof would fail, this makes the ```if (!entity.level().isClientSide())``` redundant

Reply:
Not my logic but I am happy to clean it up.